software/index.html

```diff
@@ -36,96 +37,127 @@
-                <div class="container-lg p-5">
-
-                    <h2 class="fs-1">Teaching modern software tools to the modern Purdue engineer.</h2>
-                    <p class="fs-5 text-start">
-                        Purdue IEEE is partnering with the Purdue College of Engineering to offer Software Saturdays, a FREE course available to all Purdue students!
-                    </p>
-                    <p class="fs-5 text-start">
-                        Software Saturdays introduces software development and programming frameworks to our attendees through a project-based learning experience.
-                        We introduce concepts and projects that are useful regardless of major: in the 21st century, having basic programming knowledge is vital no matter your hobbies or jobs you pursue.
-                    </p>
-                    <p class="fs-5 text-start">
-                        Sponsored by the Purdue College of Engineering, we are able to offer an official Purdue College of Engineering certification for everyone who attends!
+             <!-- Basic Info -->
+            <div class="my-3 p-5 container-fluid fade-in-left">
+                <div class="d-flex flex-column">
+                    <h2 class="fs-1 text-light">Software Development <strong class="text-secondary">Simplified</strong>.</h2>
+                    <br/>
+                    <p class="fs-4 text-start text-light">
+                         Software Satrudays is a free 8-week course, sponsered by Purdue IEEE and the College of Engineering, covering the basics of a software development framework, rotating every semester. This program is for any Purdue student who is seeking to learn a new software development framework, spruce up their resume, or is just interested in software. Additionally, everyone can earn a <strong class="purdue-text">College of Engineering</strong> certification upon completion. These look great on LinkedIn pages and resumes alike.
                     </p>
                 </div>
             </div>
 
-            <div class="my-2 p-5 container-lg">
-                <div class="row flex-wrap-reverse">
-                    <div class="col-md-10">
-                        <p class="fs-5 text-start">
-                            This semester, we are teaching Web Application Development through React.js, an ever-popular JavaScript framework. Even if you have no prior experience with React or web dev, our
-                            program will start from the basics - including HTML and CSS.
+             <!-- Semester Info -->
+            <div class="my-2 p-5 container-fluid text-light fade-in-right">
+                <div class="d-flex flex-row justify-content-between">
+                    <img src="images/react_logo.svg" width="30%" alt="ReactJS Logo" >
+                    <div class="d-flex flex-column">
+                        <h2 class="fs-1 text-light text-end">Spring 2023: <strong class="react-text">React</strong>.</h2>
```

Comment:
Right-aligned text is hard to read, keep text naturally aligned to the left. Look at the home page for an example.